requirements.txt

```diff
@@ -25,3 +25,4 @@ matplotlib<=3.2.2
 sqlalchemy
 msgpack<1.0
 pandas_gbq
+cloudpickle
```

Comment:
we should probably add `rpyc` here as well...

Reply:
Will do in separate PR